liquibase/scripts/installation.properties

```diff
@@ -0,0 +1,17 @@
+install_method=auto
+connection.url=jdbc:mysql://localhost:3306/@DBNAME@?autoReconnect=true&sessionVariables=default_storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8
+connection.driver_class=com.mysql.cj.jdbc.Driver
+connection.username=root
```

Comment:
As per the instructions you gave, i do not see how the username and password is passed over.
No wonder i got `Access denied for user 'root'@'localhost' (using password: YES)` when i tried to run this in spite of the fact that i supplied a correct username and password.

Reply:
> I just realized I missed mentioning a couple of instructions earlier, my bad!

So where are you putting the instructions?

Reply:
@ManojLL did you see the above question?

Reply:
[this is where ](https://github.com/openmrs/openmrs-core/pull/4474#issuecomment-1844327017) I put the instructions for run the script. what I forgot to mention is `chmod +x <path to script>`.

Reply:
Please do not keep instructions in a github comment.

Reply:
Did you address the above comment?

Reply:
@dkayiwa Do I need to update liquibase read me file with the instructions ?

Reply:
Yes. 😊

Reply:
@dkayiwa I added the instructions to the `Readme.md` file.